### PR TITLE
mention U.S states in usecases

### PR DESCRIPTION
**Pull request by matkoniecz**
maybe for other countries internal division is also useful from z5 - in that case it also should be mentioned

### CODE REVIEW

**Comment by math1985** [comment]
I think this applies not only to the US but to all countries with larger subdivisions (like Canada and Australia).

**Comment by matkoniecz** [comment]
Maybe - I made PR with usecase that I know that exists. I suspect that it applies also to Russia but it is only a guess and I would wait for at least single person to confirm this.

**Comment by kocio-pl** [comment]
I would rephrase it to cover all the big subdivisions (or at least mention them).

**Comment by matkoniecz** [comment]
@kocio-pl I want to avoid this as it seems likely to me that in some large countries subdivisions are not really important.

I know that in USA states are important - are we sure about other countries?

**Comment by kocio-pl** [comment]
OK, if _your_ use case is only USA, I have no problem with that. In reality currently USA, Australia and Canada subdivisions are visible (only Russia subdivisions are not visible on z4 and I don't understand why), so this is more a statement what is important than something to fix.
